presenter-in-iframe/README.md

```diff
@@ -0,0 +1,6 @@
+# Presenter in an Iframe
+
+## Description
+
+An debugging tool that deploy an hard coded (fill `src` attribute of `iframe` with `id` `thisIframe` of  `index.html:60`) presenter link in an basic web page
+the "inverse" demo of the [events-tab](../events-tab/README.md) integration.
```

Comment:
It would be nice to add a screenshot of how it is supposed to look showing the effect of e.g. the focusOn button